redisearch/autocomplete.go

```diff
@@ -40,7 +40,13 @@ func (a *Autocompleter) AddTerms(terms ...Suggestion) error {
 
 	i := 0
 	for _, term := range terms {
-		if err := conn.Send("FT.SUGADD", a.name, term.Term, term.Score); err != nil {
+
+		args := redis.Args{a.name, term.Term, term.Score}
+		if payload := term.Payload; payload != "" {
```

Comment:
why create an extra variable on the stack? just do: 

```go
if term.Payload != "" {
	args = append(args, "PAYLOAD", term.Payload)
}
```

Reply:
Fixed